packages/panzer/disc-fe/test/la_factory/CMakeLists.txt

```diff
@@ -42,31 +42,47 @@ INCLUDE_DIRECTORIES(REQUIRED_DURING_INSTALLATION_TESTING
                     ${CMAKE_CURRENT_SOURCE_DIR}
                     ${CMAKE_CURRENT_SOURCE_DIR}/../../../dof-mgr/test/dofmngr_test)
 
+SET(FILES tTpetraLinearObjFactory.cpp tTpetra_GlbEvalData.cpp UnitTest_GlobalIndexer.cpp)
+IF(PANZER_HAVE_EPETRA)
+  APPEND_SET(FILES tBlockedLinearObjFactory.cpp tBlockedTpetraLinearObjFactory.cpp
+             tEpetra_GlbEvalData.cpp)
+ENDIF(PANZER_HAVE_EPETRA)
+
+IF(PANZER_HAVE_EPETRA)
```

Comment:
No need to END and then BEGIN the same 'IF(PANZER_HAVE_EPETRA)'

Reply:
Good point. I'll update that